plugins/k8s/resoto_plugin_k8s/deferred_edges/azure.py

```diff
@@ -0,0 +1,54 @@
+from resotolib.baseresources import BaseResource
+from resotolib.graph import BySearchCriteria, ByNodeId, Graph
+from resoto_plugin_k8s.deferred_edges.utils import rgetattr
+
+
+def link_k8s_node_to_vmss_instance(graph: Graph, resource: BaseResource) -> None:
```

Comment:
As discussed: a scale set has instances. while the method title is correct, the implementation tries to attach to the scale set, not the instance.